### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by amyvarga**
Added a strapline to the logo.  Amended styling across components.

### CODE REVIEW

File: src/containers/Aya/AyaContact/AyaContact.scss [hunk 1]
```diff
@@ -13,4 +13,10 @@
 .emailAddress {
     margin-left: 5px;
 }
+ul {
```
Comment (jonreading81) [inline src/containers/Aya/AyaContact/AyaContact.scss]: Does this have any effect, its not inside a style?

File: src/components/HeroPanel/HeroPanel.js [hunk 2]
```diff
@@ -36,16 +38,16 @@ export default class HeroPanel extends Component {
...
                 <div className={type}>
                     <div className="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
 
                       <If condition={!isUndefined(title)}>
                         <h1>{title}</h1>
-                        <hr className="small" />
+                        <hr className={`${styles.hr} small`} />
                       </If>
-                      <div className="subheading">
+                      <div className={`${styles.subTitle} subheading`}>
```
Comment (jonreading81) [inline src/components/HeroPanel/HeroPanel.js]: Well done you get the idea

File: src/containers/Aya/Articles.js [hunk 3]
```diff
@@ -61,8 +62,9 @@ export default class Aya extends Component {
             {name: 'description', content: 'Aya'},
             {name: 'keywords', content: 'aya'}
           ]}
+          heroStyles={heroStyles}
           page={page}
-          articlesTitle="Articles"
+          articlesTitle=""
```
Comment (jonreading81) [inline src/containers/Aya/Articles.js]: Is this correct to be blank

Comment (amyvarga) [reply src/containers/Aya/Articles.js]: Yeah, I don't want a title as it is in the HeroPanel

File: src/containers/Aya/AyaContact/AyaContact.scss (hunk 1 above)
Comment (amyvarga) [reply src/containers/Aya/AyaContact/AyaContact.scss]: No.  Good spot, I have removed it.

**Pull request merged**